modin/pandas/dataframe.py

```diff
@@ -594,6 +594,29 @@ def append(self, other, ignore_index=False, verify_integrity=False, sort=False):
         )
         return DataFrame(query_compiler=query_compiler)
 
+    def asof(self, where, subset=None):
```

Comment:
This approach looks general enough to move to `base.py` in the `BasePandasDataset`. Then it would be implemented for both DataFrame and Series.

Reply:
I was going to do that as follow-up, but can do that here, sure.